### PR TITLE
Update Bluetooth documentation from recent issue triage

### PR DESCRIPTION
**Pull request by bdraco**
## Proposed change

- Separate the high performance adapters by chip since people still seem to buy the wrong one when buying a CSR based one is suggested.
- Add a warning that the patch files are missing for the BCM adapters
- Move the RTL adapters lacking a reset pin warning to the top of the list so its harder to miss
- Clarify that we have high availability and automatic failover with the proxies
- Clarify remote adapters and proxies are interchangeable
- Move around some warnings since they are not obvious enough to deter people from buying the "wrong" adapter
- Remove ESPHome specific configuration information now that the ESPHome docs have the same information

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/bluetooth.markdown [hunk 1]
```diff
@@ -112,6 +101,41 @@ If you experience an unreliable Bluetooth connection, installing a short USB ext
...
+Most of these adapters can hold five (5) connections at the same time.
+
+These adapters generally offer the fastest connect times and do not require additonal drivers or patch files.
+
+#### Broadcom (BCM) based adapters
+
+<div class='note warning'>
+These adapters may require additional patch files available at https://github.com/winterheart/broadcom-bt-firmware for stable operation.
+  
+There is currently no supported method to install these patch files when using Home Assistant Operating sytem.
```
Comment (bdraco) [inline source/_integrations/bluetooth.markdown]:
```suggestion
There is currently no supported method to install these patch files when using Home Assistant Operating System.
```

File: source/_integrations/bluetooth.markdown [hunk 2]
```diff
@@ -193,16 +211,18 @@ Integrations that have followed the [Best practices for library authors](https:/
 
 ## Passive Scanning
 
-Passive Scanning on Linux can be enabled in the options flow per adapter if the host system runs BlueZ 5.63 or later with experimental features enabled.
+Passive Scanning on Linux can be enabled in the options flow per adapter if the host system runs BlueZ 5.63 or later with experimental features enabled. This functionality is available with Home Assitant Operating System 9.4 and later.
```
Comment (bdraco) [inline source/_integrations/bluetooth.markdown]:
```suggestion
Passive Scanning on Linux can be enabled in the options flow per adapter if the host system runs BlueZ 5.63 or later with experimental features enabled. This functionality is available with Home Assistant Operating System 9.4 and later.
```

**Review by frenck — APPROVED** [review]

**Comment by bdraco** [comment]
Thanks.  That's going to help move some issues forward

**Comment by frenck** [comment]
Somehow I missed this in the notifications. I came across it while processing PRs for the beta release. Sorry! 😞

**Comment by bdraco** [comment]
No worries. Thanks again!

**Pull request merged**